Makefile.am

```diff
@@ -359,25 +361,27 @@ endif
 # --------------------------
 
 pcp_platform_headers = \
+	linux/PressureStallMeter.h \
+	linux/ZramMeter.h \
+	linux/ZramStats.h \
```

Comment:
Still not quite happy including Linux stuff on PCP explicitly …

Reply:
One alternative would be moving all *Meter.[ch] code into generic/ or a new meters/ ... either could make sense I think.  Might also serve to de-clutter the base htop source directory too I guess.

Either way, this is a topic for a follow-up series of commits.